common/api/ecschema-editing.api.md

```diff
@@ -1435,6 +1441,11 @@ export class SchemaItemMissing extends SchemaItemChange {
     toString(): string;
 }
 
+// @internal
+export class SchemaMerger {
```

Comment:
Who are the intended/expected consumers of the SchemaMerger class? It's exported, but marked internal, but mentioned in public changelogs, but not imported or used by anything in the core repo.

Reply:
Currently mainly iModel Transformation services.
I marked it as internal as it's a first version of that class and not complete yet. If I got you right, in that case, we should not export it in the ecschema-editing.api.md ?

Reply:
internal means this API is never intended to be used by anyone outside of the core repo.
It sounds like your new API should be marked alpha as it's eventually intended for use outside the core repo but in very early stages of development.
[How release tags work](https://www.itwinjs.org/learning/guidelines/release-tags-guidelines/).

Reply:
Alright, I mark it as beta then. It's planned to let them play with it to figure it actually meets their requirements sufficiently.